packages/editor/src/pages/Editor/store/header/selectors.ts

```diff
@@ -62,6 +63,18 @@ const getRunButton = createSelector(
     isCustomFunctionsView: boolean,
     isTrusted: boolean,
   ) => {
+    const youMustTrustBeforeRunMessageBarAction = actions.messageBar.show({
```

Comment:
Um... I guess it's an OK variable name.  But you could also have gone with `trustBeforeRunAction = ...`